client/src/components/ThreeScene/index.jsx

```diff
@@ -0,0 +1,74 @@
+import React, { Component } from 'react';
+import * as THREE from 'three';
+const STLLoader = require('three-stl-loader')(THREE);
+
+class ThreeScene extends Component {
+  componentDidMount () {
+    const width = this.mount.clientWidth;
+    const height = this.mount.clientHeight;
+
+    // ADD CAMERA
+    this.camera = new THREE.PerspectiveCamera(
+      35,
+      width / height,
+      0.1,
+      1000
+    );
+    this.camera = new THREE.PerspectiveCamera(35, width / height, 1, 15);
+    this.camera.position.set(3, 0.15, 3);
+
+    // ADD SCENE
+    this.scene = new THREE.Scene();
+    this.scene.background = new THREE.Color('#ff0000');
+
+    // ADD RENDERER
+    this.renderer = new THREE.WebGLRenderer({ antialias: true });
+    this.renderer.setPixelRatio(window.devicePixelRatio);
+    this.renderer.setSize(width, height);
+
+    this.mount.appendChild(this.renderer.domElement);
```

Comment:
You might want to check the PR's from @btzr-io here https://github.com/lbryio/lbry-desktop/pull/1558 and get his feedback as well; he's created some reusable components for 3D viewing that might give some inspiration.